esrally/chart_generator.py

```diff
@@ -1666,7 +1666,8 @@ def throttled_tasks(self):
                 #
                 # We should refactor the chart generator to make this classification logic more flexible so the user can specify
                 # which tasks / or types of operations should be used for which chart types.
-                if "target-throughput" in sub_task.params or "target-interval" in sub_task.params or sub_task.operation.type == "eql":
+                if "target-throughput" in sub_task.params or "target-interval" in sub_task.params\
```

Comment:
How about we instead use the operation type as an indicator? I propose the following operation types (in addition to `eql`):

* `search`
* `composite`
* `paginated-search`
* `scroll-search`

Reply:
hmm, just realize that if we only check for type of operation we will miss raw-requests like in https://github.com/elastic/rally-tracks/pull/179/files, I wonder if we still want to keep ` "target-throughput" in sub_task.params or "target-interval"` as well as type of operations?

Reply:
I agree, we should also consider `target-throughput` and `target-interval` as before.